bottlecap/src/traces/stats_aggregator.rs

```diff
@@ -0,0 +1,167 @@
+use datadog_trace_protobuf::pb::ClientStatsPayload;
+use prost::Message;
```

Comment:
We're using prost just to measure the size? Is there a way we can do that with whatever protobuf lib we're already using?

Reply:
ah sick, awesome ty